### PR TITLE
Apply OpenShift oAuth plugin

### PR DESCRIPTION
**Pull request by vinokurig**
Apply OpenShift oAuth plugin

### CODE REVIEW

**Comment by benoitf** [comment]
hello, should it be part of che-theia repository with plugins folder ?

**Comment by vinokurig** [comment]
It supposed to be ran inside openshift connectopr plugin side-car container. So we need it to keep the vsix file. see: https://github.com/eclipse/che-plugin-registry/pull/419

**Comment by benoitf** [comment]
@vinokurig ok I think it should be explained in the README or in description of the site

**Comment by ericwill** [comment]
@benoitf ready for re-review

File: src/che-openshift-authentication.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            return;
+        }
+    }
+
+    const isLoggedIn: boolean = await isLoggedInFunc();
+    const isAuthenticated: boolean = await che.oAuth.isAuthenticated('openshift');
+
+    if (!isLoggedIn && !isAuthenticated) {
+        const action = await theia.window.showWarningMessage(`The OpenShift plugin is not authorized, would you like to authenticate?`, 'Yes', 'No');
+        if (action == 'Yes') {
```
Comment (benoitf) [inline src/che-openshift-authentication.ts]: === ?

Comment (vinokurig) [reply src/che-openshift-authentication.ts]: fixed

**Comment by ericwill** [comment]
Do you have to do a re-release for the changes from April 9th? Otherwise LGTM.

**Comment by vinokurig** [comment]
@ericwill The release is up to date with the latest changes.

**Pull request merged**